### PR TITLE
serverErrors renamed to externalErrors

### PR DESCRIPTION
**Pull request by FermiDirak**
https://github.com/flexport/formula-one/issues/5

This PR renames all instances of serverErrors to externalErrors, and introduces name changes accordingly. This change is non-backwards compatible.

### CODE REVIEW

File: src/ErrorsHelper.js [hunk 1]
```diff
@@ -1,51 +1,52 @@
...
     shouldShowErrors,
     client: errors.client,
-    server: errors.server,
-    flattened,
+    external: errors.external,
+    flattenedErrors,
   });
 }
 
-// Using a HOC here is not possible due to a Flow bug: https://github.com/facebook/flow/issues/6903
```
Comment (zgotsch) [inline src/ErrorsHelper.js]: Why was this removed?

**Comment by zgotsch** [comment]
Thanks for doing this. I will merge this in for `0.10.0`.

File: yarn.lock [hunk 2]
```diff
@@ -9,12 +9,12 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
-"@babel/generator@^7.1.5":
-  version "7.1.5"
-  resolved "https://registry.yarnpkg.com/@babel/generator/-/generator-7.1.5.tgz#615f064d13d95f8f9157c7261f68eddf32ec15b3"
-  integrity sha512-IO31r62xfMI+wBJVmgx0JR9ZOHty8HkoYpQAjRWUGG9vykBTlGHdArZ8zoFtpUu2gs17K7qTl/TtPpiSi6t+MA==
+"@babel/generator@^7.2.2":
```
Comment (dmnd) [inline yarn.lock]: @FermiDirak why are there changes to `yarn.lock` in this PR?

**Comment by dmnd** [comment]
@zgotsch what's the process for merging this? flexport is pinned on **0.9.0-alpha.6** so it should be safe to merge to formula one master. I can update flexport to match afterwards.

**Comment by zgotsch** [comment]
0.9.0's major feature is `customChange`, its full release is blocked by #7
